umbra/umbra.h

```diff
@@ -725,6 +725,18 @@ umbra_shadow_memory_info_init(umbra_shadow_memory_info_t *info)
     info->app_size = 0;
 }
 
+DR_EXPORT
+/**
+ * Returns granularity information of the passed Umbra map.
+ *
+ * @param[in]  map              The mapping object to use.
+ * @param[out] scale            The pointer where to store the returned scale.
+ * @param[out] is_scale_down    The pointer where to store the returned flag
+ * indicating whether shadow memory is scaled down or up.
+ */
+drmf_status_t umbra_get_granularity(const umbra_map_t *map, OUT int *scale,
```

Comment:
Is there a test in tests/framework/umbra_client_*.c where we could add a call to this function?

Reply:
That is on my list b/c it is also blocking #2130

Reply:
If you could clarify: Did you want to merge this as-is and put the test in separately, or you're going to wait for more tests to be added and then re-activate this PR?

Reply:
I think it is best that we wait for tests prior to merge. I still need to see what tests exist for Umbra.

Reply:
There are a few at tests/framework/umbra_client_*.c, one of which I added recently, but I am pretty sure they are not covering the whole API.

Reply:
Should be ready now.